### DISCUSSION TITLE
Can I use Meshes without faces?

### DISCUSSION THREAD

**Discussion by hyeonLewis** [facebookresearch/pytorch3d#939] [Q&A] [answered]
Hi, I'm a student who is trying to change opendr, pyrender or others to pytorch3d renderer.

But in those renderers, faces are options and usually not used to render a mesh. 
So I tried to make faces as zero tensor or None type in pytorch3d but it just made errors.

So, my question is: Are there any ways to render a mesh without or ignore faces?

Best,

Lewis.

**Comment by bottler** [comment, marked as answer]
If there are no faces, what do you want it to look like in the output of rendering? If you want a sphere/circle at each vertex, you should use the Pointclouds object.

**Reply by hyeonLewis** [reply]
I think I misunderstood the code. Thank you for answering!
